apps/web/pages/api/recorded-daily-video.ts

```diff
@@ -105,31 +141,36 @@ async function handler(req: NextApiRequest, res: NextApiResponse) {
       },
     });
 
+    const response = await getDownloadLinkOfCalVideoByRecordingId(recordingId);
+    const downloadLinkResponse = downloadLinkSchema.parse(response);
+    const downloadLink = downloadLinkResponse.download_link;
+
+    const evt: CalendarEvent = {
+      type: booking.title,
+      title: booking.title,
+      description: booking.description || undefined,
+      startTime: booking.startTime.toISOString(),
+      endTime: booking.endTime.toISOString(),
+      organizer: {
+        email: booking.user?.email || "Email-less",
+        name: booking.user?.name || "Nameless",
+        timeZone: booking.user?.timeZone || "Europe/London",
+        language: { translate: t, locale: booking?.user?.locale ?? "en" },
+      },
+      attendees: attendeesList,
+      uid: booking.uid,
```

Comment:
Can we abstract it out in a function if not already done. This is high time I think, this is copied at so many places
<img width="570" alt="Screenshot 2023-05-09 at 10 38 15 PM" src="https://github.com/calcom/cal.com/assets/1780212/29e5925a-632c-4164-9b52-a2f876adb5d8">